lib/evss/letters/service.rb

```diff
@@ -1,9 +1,22 @@
 # frozen_string_literal: true
+require 'evss/base_service'
+
 module EVSS
   module Letters
-    class Service
-      # TODO(AJD): only mock letter service for now...
-      def initialize(_headers)
+    class Service < EVSS::BaseService
+      BASE_URL = "#{Settings.evss.url}/wss-lettergenerator-services-web/rest/letters/v1"
+
+      def get_letters
+        raw_response = get ''
+        EVSS::Letters::LettersResponse.new(raw_response)
+      end
+
+      def get_letter_for_user_by_type(type)
+        get type
+      end
+
+      def self.breakers_service
+        BaseService.create_breakers_service(name: 'EVSS/Letters', url: BASE_URL)
```

Comment:
The `dev` urls we were given have the same root of `https://csraciapp6.evss.srarad.com/` but different paths of `/wss-education-services-web/rest/education/chapter33/v1` & `/wss-lettergenerator-services-web/rest/letters/v1/commissary` for GI Bill and Letters respectively.  Do we need 2 separate breakers services or just one?